static/src/javascripts/projects/commercial/modules/dfp/Advert.js

```diff
@@ -16,7 +16,7 @@ type Resolver = (x: boolean) => void;
  * One size       - `data-mobile="300,50"`
  * Multiple sizes - `data-mobile="300,50|320,50"`
  */
-const createSizeMapping = (attr: string): Array<AdSize> =>
+const createSizeMapping = (attr: string): Array<SingleSize> =>
```

Comment:
I think `SingleSize` initially seems verbose as a type name. It's both grammatically singular and explicitly qualified as singular. But I can see an issue maybe caused by `GeneralSize`. Maybe we could call
`GeneralSize` -> `AdSize`?